hoot-core/src/main/cpp/hoot/core/io/OsmApiChangesetElement.h

```diff
@@ -240,30 +243,30 @@ class XmlWay : public XmlElement
   QVector<long> _nodes;
 };
 /** Handy typedef for way shared pointer */
-typedef std::shared_ptr<XmlWay> XmlWayPtr;
+typedef std::shared_ptr<ChangesetWay> ChangesetWayPtr;
 
 /** Simplified changeset relation member abstraction */
-class XmlMember
+class ChangesetRelationMember
 {
 public:
   /**
-   * @brief XmlMember constructor
+   * @brief ChangesetRelationMember constructor
    * @param member Member attributes
    * @param idMap ID to ID Map for updated IDs
    */
-  XmlMember(const QXmlStreamAttributes& member, ElementIdToIdMap* idMap);
+  ChangesetRelationMember(const QXmlStreamAttributes& member, ElementIdToIdMap* idMap);
   /**
    * @brief isNode/Way/Relation
    * @return true if relation member is node/way/relation
    */
-  bool isNode()     { return _type == "node"; }
-  bool isWay()      { return _type == "way"; }
-  bool isRelation() { return _type == "relation"; }
+  bool isNode()     { return _type == ElementType::Node; }
+  bool isWay()      { return _type == ElementType::Way; }
+  bool isRelation() { return _type == ElementType::Relation; }
```

Comment:
Reusing the `ElementType::Type` enumeration instead of a string to reduce memory usage.  It is a small gain but was easy to implement.